patches/extra/ungoogled-chromium/add-flag-to-hide-side-panel-button.patch

```diff
@@ -1,16 +1,6 @@
---- a/chrome/browser/ui/views/frame/browser_view.cc
-+++ b/chrome/browser/ui/views/frame/browser_view.cc
-@@ -955,6 +955,7 @@ BrowserView::BrowserView(std::unique_ptr
-         AddChildView(std::make_unique<ContentsSeparator>());
-     side_panel_coordinator_ = std::make_unique<SidePanelCoordinator>(this);
-   } else {
-+   if (!base::CommandLine::ForCurrentProcess()->HasSwitch("hide-sidepanel-button"))
```

Comment:
Is this hunk no longer needed?

Reply:
I stand corrected. The hunk I was suspecting is in fact untouched: https://github.com/ungoogled-software/ungoogled-chromium/commit/515cd084ce0889c4b84002eb9c6d1eb7c8fa77d2

@gz83 Could you elaborate on what could be broken?

Reply:
That change was for the previous iteration of the sidepanel.  Now that they've transitioned to the unified sidepanel only the change @PF4Public linked is needed, which is why the previous code was removed.